<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdicionarEsferasSistematicamenteComAjusteDaCaixaDialog</class>
 <widget class="QDialog" name="AdicionarEsferasSistematicamenteComAjusteDaCaixaDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adição Sistematizada de Esferas</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Fração de Vazio (%):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textFracaoVazia">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemover">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>331</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Remover</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdicionar">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>330</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>390</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableEspecificacao">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>48</y>
     <width>491</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>351</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Lado da Caixa:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolumeFaseSolida">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>337</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>337</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Volume acumulado da fase sólida:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLadoDaCaixa">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnAdicionar</sender>
   <signal>clicked()</signal>
   <receiver>AdicionarEsferasSistematicamenteComAjusteDaCaixaDialog</receiver>
   <slot>adicionarDescricao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemover</sender>
   <signal>clicked()</signal>
   <receiver>AdicionarEsferasSistematicamenteComAjusteDaCaixaDialog</receiver>
   <slot>removerDescricao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>AdicionarEsferasSistematicamenteComAjusteDaCaixaDialog</receiver>
   <slot>sair()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>AdicionarEsferasSistematicamenteComAjusteDaCaixaDialog</receiver>
   <slot>adicionarEsferas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>adicionarDescricao()</slot>
  <slot>removerDescricao()</slot>
  <slot>adicionarEsferas()</slot>
  <slot>sair()</slot>
 </slots>
</ui>
